Content.Server/Kitchen/EntitySystems/ReagentGrinderSystem.cs

```diff
@@ -39,7 +39,8 @@ public override void Initialize()
             base.Initialize();
 
             SubscribeLocalEvent<ReagentGrinderComponent, ComponentInit>(OnComponentInit);
-            SubscribeLocalEvent<ReagentGrinderComponent, PowerChangedEvent>((_, component, _) => EnqueueUiUpdate(component));
+            SubscribeLocalEvent<ReagentGrinderComponent, PowerChangedEvent>((_, component, _) =>
+                EnqueueUiUpdate(component));
             SubscribeLocalEvent<ReagentGrinderComponent, InteractHandEvent>(OnInteractHand);
```

Comment:
This is another class where I have no idea what's even changing (turns out it was lines 68-70) as your IDE put christmas onto github.